resources/intro-to-observables.md

```diff
@@ -0,0 +1,123 @@
+# Observables: introduction
+
+Since JavaScript is single-threaded, we often run into situations that involve asynchronous data flows.
+
+## What are asynchronous data flows again? 
```

Comment:
What do people think, do we need this section??

Reply:
Yeah totally, this is a complete intro, and it's a good idea to describe what the problem is that needs to be solved

Reply:
👍